Fundamentals/Setup/Upgrading/version-specific.md

```diff
@@ -94,10 +97,31 @@ To:
 "$schema": "./appsettings-schema.json",
 ```
 
+
 :::note
 To upgrade to Umbraco 10, your database needs to be at least on Umbraco 8.18.
 :::
 
+### Upgrade of any publicly hosted environment
+
+When the upgrade is completed and tested, and prior to deploying to any publicly accessible environment, you should consider the following:
+
+1. You will want to ensure you have backups for both the datbaase and the file system for rollback.
+2. You should stop the site so it is not accessable during the upgrade process.
+3. You should delete the relevant folders from the filesystem prior to deploying.
+  - `/wwwroot/umbraco`
+  - `/umbraco/PartialViewMacros`
+  - `/umbraco/UmbracoBackOffice`
+  - `/umbraco/UmbracoInstall`
+  - `/umbraco/UmbracoWebsite`
+  - `/umbraco/config/lang`
+  - `/umbraco/config/appsettings-schema.json`
+4. Deploy the site how you normally would to your public facing environment 
+5. Start the site. At this point it will launch and upgrade the database, after which the site should become accessible and your upgrade is complete. 
+6. Remember to check the logs for any errors which may have occurred during the upgrade process. 
+:::
```

Comment:
This `:::` is closing a note. But I don't see it started anywhere? 🤔 

Did you mean to put this part in a note/tip, or should these three colons be taken out?